source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -163,15 +162,12 @@ def get_timeout(self, obj):
     def get_backend_class(self, obj):
         return obj.get('backend_class', None)
 
-    def get_matrix_type(self, obj):
-        return obj.get('matrix_type', None)
+    def get_lapse_time_matrix_to_retry(self, obj):
+        return obj.get('lapse_time_matrix_to_retry', None)
```

Comment:
To remove, no ?

Reply:
mandatory du to https://github.com/CanalTP/navitia/pull/3662#discussion_r806014886